README.md

```diff
@@ -90,10 +90,13 @@ The [Helm chart](https://artifacthub.io/packages/helm/metrics-server/metrics-ser
 
 ## Security context
 
-Metrics Server requires the `CAP_NET_BIND_SERVICE` capability in order to bind to a privileged ports as non-root.
-If you are running Metrics Server in an environment that uses PSPs or other mechanisms to restrict pod capabilities, ensure that Metrics Server is allowed
-to use this capability.
-This applies even if you use the `--secure-port` flag to change the port that Metrics Server binds to to a non-privileged port.
+Metrics Server may require the `CAP_NET_BIND_SERVICE` capability, in order to bind to a privileged port as non-root.
+As such, Kubernetes versions [prior to v1.23.0](https://github.com/kubernetes/kubernetes/commit/3d620192d95dbacf5ad9c18755576b15047a8d10) require this capability.
```

Comment:
This link points to K8s release which maintain their own Metrics Server release. This configuration is independent of Metrics Server and should not be referenced here.

Reply:
Sure I can do that, I was just adding it as context anyway

Reply:
Mm just before I do that, @serathius if you could please read [my comment above](https://github.com/kubernetes-sigs/metrics-server/pull/921/files#r786321873), explaining my original rationale. I can appreciate not wanting to avoid leaking concerns of other projects. If possible I would like to better inform users of the problem case I've described (and ran into, myself). The difficulty with this is that the problem case pertains to how Metrics Server is used, and that this documentation is what users will find when they are attempting to debug this particular issue.